test/hotspot/jtreg/vmTestbase/nsk/jdi/EventRequest/disable/disable001.java

```diff
@@ -292,14 +244,10 @@ private void testRun()
 
         log2("      received: ClassPrepareEvent for debuggeeClass");
 
-        String bPointMethod = "methodForCommunication";
-        String lineForComm  = "lineForComm";
-
-        ThreadReference   mainThread = debuggee.threadByNameOrThrow("main");
-
-        BreakpointRequest bpRequest = settingBreakpoint(mainThread,
-                                             debuggeeClass,
-                                            bPointMethod, lineForComm, "zero");
+        BreakpointRequest bpRequest =
+            setupBreakpointForCommunication(debuggeeClass);
+        bpRequest.disable();
+        bpRequest.setSuspendPolicy(EventRequest.SUSPEND_ALL);
```

Comment:
Could you please add a comment that setupBreakpointForCommunication creates request with SUSPEND_EVENT_THREAD policy, need to change it to SUSPEND_ALL

Reply:
Done.